docs/guides/src/main/server/db.adoc

```diff
@@ -72,50 +75,65 @@ Database schema only accounts for Unicode strings in the following special field
 
 * Descriptions of objects
 
-Otherwise, characters are limited to those contained in database encoding which is often 8-bit. However, for some database systems, it is possible to enable UTF-8 encoding of Unicode characters and use full Unicode character set in all text fields. Often, this is counterbalanced by shorter maximum length of the strings than in case of 8-bit encodings.
+Otherwise, characters are limited to those contained in database encoding, which is often 8-bit. However, for some database systems, you can enable UTF-8 encoding of Unicode characters and use the full Unicode character set in all text fields. For a given database, this choice might result in a shorter string length than the maximum string length supported by 8-bit encodings.
+
+=== Configuring Unicode support for an Oracle database
```

Comment:
This section is just a copy/paste from the old documentation. @hmlnarik could you take a look at the suggested changes here?

Reply:
@hmlnarik Do you have any comment on this change?

Reply:
I believe all my comments have been addressed by now. Thank you